exercises/concept/card-games/lists_test.py

```diff
@@ -12,337 +12,153 @@
 )
 
 
-class TestToRounds(unittest.TestCase):
+class CardGamesTest(unittest.TestCase):
 
     @pytest.mark.task(taskno=1)
-    def test_instructions_example(self):
-        round_number = 27
-        want = [27, 28, 29]
-        got = get_rounds(round_number)
-
-        self.assertEqual(
-            want,
-            got,
-            msg=f'Expected {want} but got an incorrect result: {got!r}'
-        )
-
-    @pytest.mark.task(taskno=1)
-    def test_zero(self):
+    def test_round_number_zero(self):
         round_number = 0
         want = [0, 1, 2]
-        got = get_rounds(round_number)
 
-        self.assertEqual(
-            want,
-            got,
-            msg=f'Expected {want} but got an incorrect result: {got!r}'
+        self.assertEqual(want, get_rounds(round_number),
+            msg=f'Expected {want} but got an incorrect result.'
         )
 
     @pytest.mark.task(taskno=1)
-    def test_random_int(self):
+    def test_random_int_for_round_number(self):
         round_number = random.randint(0, 100)
         want = [round_number + i for i in range(3)]
-        got = get_rounds(round_number)
 
-        self.assertEqual(
-            want,
-            got,
-            msg=f'Expected {want} but got an incorrect result: {got!r}'
+        self.assertEqual(get_rounds(round_number), want,
+            msg=f'Expected {want} but got an incorrect result.'
         )
 
-
-class TestConcatenateRounds(unittest.TestCase):
-
     @pytest.mark.task(taskno=2)
-    def test_empty(self):
+    def test_concatenate_empty_rounds(self):
         rounds_1 = []
         rounds_2 = []
         want = []
 
-        self.assertEqual(concatenate_rounds(rounds_1, rounds_2),
-                         want,
-                         msg=f'Expected {want} but got an incorrect result.'
-                         )
+        self.assertEqual(concatenate_rounds(rounds_1, rounds_2), want,
+            msg=f'Expected {want} but got an incorrect result.'
+        )
 
     @pytest.mark.task(taskno=2)
-    def test_other(self):
+    def test_concatenate_other_rounds(self):
         rounds_1 = [1, 2, 3]
         rounds_2 = [4, 5, 6]
         want = [1, 2, 3, 4, 5, 6]
 
-        self.assertEqual(concatenate_rounds(rounds_1, rounds_2),
-                         want,
-                         msg=f'Expected {want} but got an incorrect result.'
-                         )
-
-
-class TestListContainsRound(unittest.TestCase):
+        self.assertEqual(concatenate_rounds(rounds_1, rounds_2), want,
+            msg=f'Expected {want} but got an incorrect result.'
+            )
```

Comment:
Once modifying these files, it might be worth running it through `blacken` for formatting. This `)` is oddly indented

Reply:
We have not yet decided that `black` -- or the hanging indent is gong to be the format.  This was only about the test names.